src/forms/FormNav/FormNav.styled.tsx

```diff
@@ -125,6 +125,7 @@ export const FormNavRow = styled.div`
 
 export const FormNavWrapper = styled.div`
   background: white;
+  margin-top: 30px;

```

Comment:
On big desktop this is leaving a gap between the top component and the sidenav, but on the xd they look together. 
![image](https://user-images.githubusercontent.com/23409284/176061354-ae033dcc-5b22-41b8-a5cd-8abcb2c667ab.png)

If this affects other places let's just keep the margin-top 30px on all sizes except big desktop (if it doesn't affect then I'd recommend removing it).